src/membrane-keypad-element.ts

```diff
@@ -26,27 +33,46 @@ export class MembraneKeypadElement extends LitElement {
 
   render() {
     const fourColumns = !this.threeColumns;
-    const columnWidth = 14.647;
+    const columnWidth = 15;
     const width = fourColumns ? 70.336 : 70.336 - columnWidth;
 
     return html`
       <style>
         text {
-          fill: #c2e4fd;
+          fill: #dfe2e5;
           user-select: none;
         }
 
         g[tabindex] {
           cursor: pointer;
         }
 
-        g[tabindex]:focus {
-          fill: #4e50d7;
+        g[tabindex]:focus,
+        g[tabindex]:active {
           stroke: white;
           outline: none;
         }
 
+        .blueKey:focus,
+        .redKey:focus {
```

Comment:
We use "kebab-case" for CSS class name in [other parts of the project](https://github.com/wokwi/wokwi-elements/blob/1467d953010919da5d51fc2bfa692a04ae7e8983/src/lcd1602-element.ts#L29), so let's change these to `blue-key` and `red-key` to be consistent